src/domain/__tests__/GetNotificationDetailRecord.test.ts

```diff
@@ -0,0 +1,18 @@
+import { makeGetNotificationDetailRecord } from '../GetNotificationDetailRecord';
+import * as data from './data';
+
+describe('makeGetNotificationDetailRecord', () => {
+  describe('200 responses', () => {
+    it('should not return the notificationRequestId property', () => {
+      const actual = makeGetNotificationDetailRecord(data.makeTestSystemEnv())({
+        apiKey: data.apiKey.valid,
+        iun: data.aIun.valid,
+      })([
+        data.newNotificationRecord,
+        data.checkNotificationStatusRecordAccepted,
+        data.getNotificationDetailRecordAcceptedWithTimeline,
+      ]);
+      expect(actual.output.returned).not.toHaveProperty('notificationRequestId');
```

Comment:
```suggestion
      expect(actual.output.returned).toEqual(expect.not.objectContaining({ notificationRequestId: expect.anything() }));
      expect(actual.output.returned).toEqual(expect.objectContaining({ iun: data.aIun.valid }));
```
what do you think of something like this?